contracts/truefi2/TrueRateAdjuster.sol

```diff
@@ -74,10 +74,22 @@ contract TrueRateAdjuster is ITrueRateAdjuster, UpgradeableClaimable {
         return combinedRate(poolBasicRate(pool), creditScoreAdjustmentRate(score));
     }
 
+    function proFormaRate(
+        ITrueFiPool2 pool,
+        uint8 score,
+        uint256 amount
+    ) external override view returns (uint256) {
+        return combinedRate(proFormaPoolBasicRate(pool, amount), creditScoreAdjustmentRate(score));
+    }
+
     function poolBasicRate(ITrueFiPool2 pool) public override view returns (uint256) {
         return min(riskPremium.add(securedRate(pool)).add(utilizationAdjustmentRate(pool)), MAX_RATE_CAP);
     }
 
+    function proFormaPoolBasicRate(ITrueFiPool2 pool, uint256 amount) public view returns (uint256) {
+        return min(riskPremium.add(securedRate(pool)).add(proFormaUtilizationAdjustmentRate(pool, amount)), MAX_RATE_CAP);
```

Comment:
This method does exactly the same thing, the one above does. Maybe we can extract some helper method for that?